packages/block-library/src/latest-posts/index.php

```diff
@@ -99,6 +99,14 @@ function register_block_core_latest_posts() {
 					'type'    => 'number',
 					'default' => 5,
 				),
+				'displayPostContent' => array(
+					'type'    => 'boolean',
+					'default' => false,
+				),
+				'excerptLength' => array(
+					'type'    => 'number',
+					'default' => 55,
```

Comment:
Is this `55` an arbitrary setting or does it mirror a default already in WP Core?

Reply:
It mirrors other blocks like RSS that have an excerpt slider